site/_docs/reference.md

```diff
@@ -2003,13 +2003,35 @@ Note:
 
 | Operator syntax                                   | Description
 |:------------------------------------------------- |:-----------
-| JSON_TYPE(value)                                  | Returns a string indicating the type of a JSON **value**. This can be an object, an array, or a scalar type
-| JSON_DEPTH(value)                                 | Returns a integer indicating the depth of a JSON **value**. This can be an object, an array, or a scalar type
-| JSON_PRETTY(value)                                | Returns a pretty-printing of JSON **value**.
+| JSON_TYPE(value)                                  | Returns a string value indicating the type of a JSON **value**, *value* is in character string type
+| JSON_DEPTH(value)                                 | Returns a integer value indicating the depth of a JSON **value**, *value* is in character string type
```

Comment:
Elsewhere in this doc we define variables and their type, e.g. "In the following: *geom* is a GEOMETRY". You could define *jsonValue* similarly, and therefore you can be more concise when specifying function.

Reply:
Thank you for taking a look @julianhyde!
I've added type definition for `jsonValue` and `path` right below the parent title "JSON Functions".